src/components/common/Layout/components/Account/components/AccountModal/components/LinkedAddress.tsx

```diff
@@ -27,13 +27,14 @@ import useDisconnect from "../hooks/useDisconnect"
 
 type Props = {
   address: string
+  isPrimary: boolean
```

Comment:
I think you can move the `addresses.indexOf(address) === 0` logic inside this file and remove this prop, it would be a cleaner solution in my opinion (just like you did inside `AccountModal`)